site/resources.md

```diff
@@ -49,31 +49,21 @@ Here you will find external resources about Velero, such as videos, podcasts, an
 
 ## Blog posts
 
-* How to Make Up for Kubernetes’ Disaster Recovery Shortfalls - by Carlisia Campos
+* [How to Make Up for Kubernetes’ Disaster Recovery Shortfalls - by Carlisia Campos](https://thenewstack.io/how-to-make-up-for-kubernetes-disaster-recovery-shortfalls/)
 
-    https://thenewstack.io/how-to-make-up-for-kubernetes-disaster-recovery-shortfalls/
+* [Project Velero 1.2 on OpenShift 3.11 - Backup and Restore (and backup to NFS) - by Rafael Brito](http://www.rafaelbrito.com/2019/11/project-velero-12-on-openshift-311.html)
 
-* Project Velero 1.2 on OpenShift 3.11 - Backup and Restore (and backup to NFS) - by Rafael Brito
+* [Backing Up Your Kubernetes Applications with Velero v1.1 - by Joe Mann](https://mannimal.blog/2019/10/04/backing-up-your-kubernetes-applications-with-velero-v1-1/)
 
-    http://www.rafaelbrito.com/2019/11/project-velero-12-on-openshift-311.html
+* [Using Velero for K8s Backup and Restore of CSI Volumes - by Myles Gray](https://blah.cloud/automation/using-velero-for-k8s-backup-and-restore-of-csi-volumes/)
 
-* Backing Up Your Kubernetes Applications with Velero v1.1 - by Joe Mann
-
-    https://mannimal.blog/2019/10/04/backing-up-your-kubernetes-applications-with-velero-v1-1/
-
-* Using Velero for K8s Backup and Restore of CSI Volumes - by Myles Gray
-
-    https://blah.cloud/automation/using-velero-for-k8s-backup-and-restore-of-csi-volumes/
-
-* "Kubernetes Backup: Improve already awesome Velero with OpenEBS" - by Harshita Sharma
-
-    https://blog.mayadata.io/openebs/suggesting-ways-to-improve-already-awesome-velero
+* ["Kubernetes Backup: Improve already awesome Velero with OpenEBS" - by Harshita Sharma](https://blog.mayadata.io/openebs/suggesting-ways-to-improve-already-awesome-velero)
 
 * Two great blog posts by community member Imran Pochi:
+    * [Backup and Restore of Kubernetes Applications using Heptio’s Velero with Restic and Rook-Ceph as the storage provider](https://blog.kubernauts.io/backup-and-restore-of-kubernetes-applications-using-heptios-velero-with-restic-and-rook-ceph-as-2e8df15b1487)
```

Comment:
Maybe add spaces in between the bullets here, so the list is formatted with the same double spacing throughout. Right now the Cormac Hogan link looks too close to the indented bullets

Reply:
@a-mccarthy I see what you are saying. But That is just how that markdown is being rendered. Adding new lines didn't seem to solve the spacing 🙁

Reply:
I think adding two spaces on the new line might help here.

Reply:
I don't know if that'll help. But I've did it anyway. PTAL

Reply:
![image](https://user-images.githubusercontent.com/22872138/86418482-1d82b900-bc85-11ea-82f4-083eae247772.png)

Doesn't seem like it helped

![image](https://user-images.githubusercontent.com/22872138/86418505-33907980-bc85-11ea-8286-0383865bb923.png)

Reply:
I looked into it, it's because of some strangeness in how second layer lists are handled. If you can move that section down to the bottom so we can skip the weirdness for now, then I can dig into what needs to be changed later.

Reply:
@ashish-amarnath have you tried adding spaces in between all the bullet points, even the second layer lists?

Reply:
@a-mccarthy adding new-lines between bullet points didn't work. Adding whitespace after `*` is not valid markdown and this is what happens when I do that
![image](https://user-images.githubusercontent.com/22872138/86939856-ef72fe00-c0f6-11ea-8964-b8491f3cdf93.png)

I  like @jonasrosland's idea of moving the weirdness to the bottom! That should def make this go away!

Reply:
@a-mccarthy and @jonasrosland PTAL! https://deploy-preview-2667--velero.netlify.app/resources/